### DISCUSSION TITLE
ScriptReference/Rigidbody2D-freezeRotation

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#631] [Notes]
# ScriptReference/Rigidbody2D-freezeRotation

https://docs.unity3d.com/ScriptReference/Rigidbody2D-freezeRotation.html

**Comment by qwe321qwe321qwe321** [comment]
[Rigidbody2D with FreezePosition has weird huge rotating on two circles contact each other.](https://forum.unity.com/threads/rigidbody2d-with-freezeposition-has-weird-huge-rotating-on-two-circles-contact-each-other.1186549) (Fixed in 2020.3)
